server/src/tsp-client.ts

```diff
@@ -102,7 +102,7 @@ export class TspClient {
         this.cancellationPipeName = tempy.file({ name: 'tscancellation' } as any);
         args.push('--cancellationPipeName', this.cancellationPipeName + '*');
         this.logger.info(`Starting tsserver : '${tsserverPath} ${args.join(' ')}'`);
-        this.tsserverProc = cp.spawn(tsserverPath, args);
+        this.tsserverProc = cp.spawn(process.execPath, args);
```

Comment:
I'm not sure why but `cp.fork(tsserverPath, args)` (and changing args to not include the `tsserverPath` crashes on the next line (the readline interface creation). So I kept this as `cp.spawn`